os/binfmt/libelf/libelf_read.c

```diff
@@ -108,6 +108,46 @@ static inline void elf_dumpreaddata(FAR char *buffer, int buflen)
  * Public Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: bin_read_header
+ *
+ * Description:
+ *   Read 'readsize' bytes from the object file.  The data is
+ *   read into 'buffer'.
+ *
+ * Returned Value:
+ *   0 (OK) is returned on success and a negated errno is returned on
```

Comment:
Sometimes, read bytes is returned.
Please modify the explanation.

Reply:
It will be less than 0 which mean error case, not actual bytes read will be return ever.